src/package/subscriberPackageVersion.ts

```diff
@@ -145,20 +146,20 @@ export class SubscriberPackageVersion {
   public static async installStatus(
     connection: Connection,
     packageInstallRequestOrId: string | PackagingSObjects.PackageInstallRequest,
-    installationKey?: string,
+    installationKey?: string | undefined | Nullable<string>,
```

Comment:
If installationKey is defined, it must be a string.

Reply:
Will look